@here/harp-mapview/lib/MapView.ts

```diff
@@ -13,6 +13,7 @@ import { IMapAntialiasSettings, IMapRenderingManager, MapRenderingManager } from
 import { ConcurrentDecoderFacade } from "./ConcurrentDecoderFacade";
 import { CopyrightInfo } from "./CopyrightInfo";
 import { DataSource } from "./DataSource";
+import { ElevationRangeSource } from "./ElevationRangeSource";
```

Comment:
How this even passed CI ?
It should be rejected as not-used symbol.

Reply:
Hm, strange. It was also not cached when I ran tslint locally. I will do a follow-up commit to fix it.

Reply:
I'm already looking at CI scripts to check what have happened.